scripts/startup_api.sh

```
@@ -5,22 +5,4 @@ GUNICORN_CMD_ARGS=${GUNICORN_CMD_ARGS:-""} # ex: --log-config app/log.conf
-    -n albert-worker@%h \
-    -Q ${CELERY_QUEUES} \
-    -c ${CELERY_CONCURRENCY} \
-    --loglevel=${CELERY_LOG_LEVEL:-INFO} ${CELERY_EXTRA_ARGS:-""} &
-fi
-
-exec gunicorn api.main:app \
-    --worker-class uvicorn.workers.UvicornWorker \
-    --bind 0.0.0.0:8000 \
-    $GUNICORN_CMD_ARGS  
-
+exec gunicorn api.main:app --worker-class uvicorn.workers.UvicornWorker --bind 0.0.0.0:8000 $GUNICORN_CMD_ARGS  
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)